docs/how-to/default-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default profile
+legacyUrl: https://docs.ipfs.io/guides/examples/default-profile/
+description: Hands-on guide for rolling back to the default profile
+---
+
+# Configure a default profile and switching between profiles
+
+---
+### What is a profile
+A profile can be used to tweak the performance of your IPFS node. If you want to have a faster, more performant datastore for your IPFS node, you can use a profile like `badgerds`, but also it requires more memory. If you're running your ipfs node on a low-power device (like a raspberry-pi), you can use a profile like `low-power`. Profiles have been developed to customize your ipfs node to perform best under given conditions.
+
+### Default Profile
+By default, IPFS uses `flatfs` profile. This is the most battle-tested and reliable datastore. You can verify this by inspecting your IPFS config file, it's usually located at `$IPFS_PATH/config`. If your `$IPFS_PATH` is not set, then it'll be located at `$HOME/.ipfs/config`.
```

Comment:
Replace this paragraph with a [list of the available profiles.](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md).

```suggestion
```